.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,40 @@
+# template from https://github.com/JamesIves/github-pages-deploy-action/tree/dev
+
+name: CI
+
+run-name: Deployment Triggered - ${{ github.event.workflow_run.conclusion == 'success'}}
+
+on:
+  workflow_run:
+    workflows: ["Run tests"]
+    types:
+      - completed
+
+jobs:
+  build-and-deploy:
+    #concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧
+        run: |
+          npm install
+          npm run-script build
```

Comment:
I believe that for the demo site, we want a `npm run-script build:prod` or probably rather `build:fr`.

Reply:
Although I discovered the other day that there's a bug in package.json, 
```
"build:fr": "npm run build -- --configuration=fr-production",`
```
should be `--configuration=fr,production` with a comma between `fr` and `production`. If you use `build:fr` you'll have to fix that bug at the same time. We should fix it regardless, really.

Reply:
This was what I was unsure about. I did have build:prod initially but it didn't look right when it spun up. And then I saw that it at least looked wrong in the same way that the one thats currently up on the wordweaver gh-pages does.